bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -19,22 +19,95 @@ metadata:
+    provider: Red Hat
+    repository: https://github.com/openshift/hw-event-proxy
+    support: Red Hat
   name: hw-event-proxy-operator.v4.12.0
-  namespace: placeholder
+  namespace: openshift-bare-metal-events
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: HardwareEvent
+    - description: HardwareEvent is the Schema for the hardwareevents API
+      displayName: Hardware Event
+      kind: HardwareEvent
       name: hardwareevents.event.redhat-cne.org
+      resources:
+      - kind: Deployment
+        name: ""
+        version: apps/v1
+      - kind: Namespace
+        name: ""
+        version: v1
+      specDescriptors:
+      - displayName: Log Level
+        path: logLevel
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Message Parser Timeout
+        path: msgParserTimeout
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Node Selector
+        path: nodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: 'TransportHost format is <protocol>://<transport-service>.<namespace>.svc.cluster.local:<transport-port>"
+          Example HTTP transport: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
+          Example AMQP transport: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"'
+        displayName: Transport Host
+        path: transportHost
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      statusDescriptors:
+      - description: LastSynced time of the custom resource
+        displayName: Last Synced
+        path: lastSyncTimestamp
       version: v1alpha1
-  description: Hw Event Proxy Operator description. TODO.
-  displayName: Hw Event Proxy Operator
+  description: |
+    ## Funtionality
+    Install and manage a hardware event delivery framework.
+    ## Installation
+    To start using Hardware event proxy, install the operator and then create the following HardwareEvent:
+    ```
+    apiVersion: "event.redhat-cne.org/v1alpha1"
+    kind: "HardwareEvent"
+    metadata:
+      name: "hardware-event"
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
+
+    ```
+    This operator needs a secret to be created to access Redfish
+    Message Registry.
+    The secret name and spec the operand expects are under the same namespace
+    the operand is deployed on. For example:
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: redfish-basic-auth
+    type: Opaque
+    stringData:
+      username: <bmc_username>
+      password: <bmc_password>
+      # BMC host DNS or IP address
+      hostaddr: <bmc_host_ip_address>
+    ```
+  displayName: Bare Metal Event Relay Operator
```

Comment:
do we want to use the `Bare Metal Event Relay` displayName upstream?.. we have been using `Hardware Event Proxy Operator` here so far..

Reply:
updated